.eslintrc

```diff
@@ -0,0 +1,57 @@
+{
+  "extends": ["surikat/es6", "prettier"],
+  "plugins": ["@typescript-eslint"],
+  "parser": "@typescript-eslint/parser",
+  "rules": {
+    "arrow-body-style": "off",
+    "new-cap": "off",
+    "no-param-reassign": "off",
+    "no-shadow": "off",
+    "no-unused-expressions": "warn",
+    "no-unused-vars": "off",
+    "no-use-before-define": "off",
+    "no-useless-constructor": "off",
+    "import/no-cycle": "off",
+    "import/order": "off",
+    "prefer-destructuring": "off",
+    "@typescript-eslint/no-shadow": "error",
+    "@typescript-eslint/no-unused-expressions": "error",
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-use-before-define": ["error", { "functions": false }],
+    "@typescript-eslint/no-useless-constructor": "error",
+
+    //Temporary disabled after eslint8 upgrade
+    "import/prefer-default-export": "warn",
+    "no-else-return": "warn",
+    "no-promise-executor-return": "warn",
+    "import/no-duplicates": "warn",
+    "import/first": "warn",
+    "import/no-useless-path-segments": "warn",
+    "default-param-last": "warn",
+    "import/named": "warn",
+    "prefer-promise-reject-errors": "warn",
+    "no-lonely-if": "off",
+    "no-undef-init": "warn",
+    "no-await-in-loop": "warn",
+    "no-async-promise-executor": "warn",
+    "no-restricted-syntax": "warn",
+    "no-useless-return": "warn",
+    "operator-assignment": "warn",
+    "spaced-comment": "warn",
+    "no-useless-computed-key": "warn",
+    "no-restricted-exports": [
+      "error",
+      {
+        "restrictedNamedExports": ["then"]
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "no-undef": "off"
+      }
+    }
+  ]
```

Comment:
A lot of this is unnecessary. We don't want to add unnecessary customization. Remove all customization and add only if needed.